lib/global-styles.php

```diff
@@ -107,7 +107,7 @@ function_exists( 'gutenberg_is_edit_site_page' ) &&
 	}
 	$consolidated = WP_Theme_JSON_Resolver::get_merged_data( $settings, $origin );
 
-	if ( 'mobile' === $context ) {
+	if ( WP_Theme_JSON_Resolver::theme_has_support() && 'mobile' === $context ) {
```

Comment:
What if we only set context to `mobile` if the theme has support?

Reply:
That's even better, I'll make the change, thanks! 👍

Reply:
Updated https://github.com/WordPress/gutenberg/pull/31844/commits/cf9486eba8fc809026a5a72490d169998432ddbc